sensor_msgs_py/CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+Any contribution that you make to this repository will
+be under the 3-Clause BSD License, as dictated by that
+[license](https://opensource.org/licenses/BSD-3-Clause).
```

Comment:
just checking, only this one is NOT Apache 2 License? and that is correct, right?

Reply:
Correct, only `sensor_msgs_py` is BSD license.  Everything else in this repository is Apache 2.0.